subprojects/shipkit/shipkit.gradle

```diff
@@ -8,6 +8,9 @@ apply plugin: 'com.gradle.plugin-publish'
 
 apply from: "$rootDir/gradle/upgrade-downstream.gradle"
 
+//TODO: WW uncomment when GradlePluginArtifactUrlResolver is released
+//apply plugin: 'org.shipkit.compare-publications'
```

Comment:
Why don't we fold it under 'org.shipkit.gradle-plugin'? It's already very opinionated by design.